app/js/books_page/books_page_ctrl.js

```diff
@@ -1,12 +1,13 @@
 'use strict';
 
-class BooksPageCtrl {
-  constructor() {
-    this.searchText = '';
-  }
+function BooksPageCtrl() {
+  let vm = this;
```

Comment:
I would go with `ctrl`. It's easier to understand, it matches with what it is and we are already using it in other projects.